.github/workflows/main.yml

```diff
@@ -70,6 +70,35 @@ jobs:
           file: build/*/*.dmg
           tag: ${{ github.ref }}
           file_glob: true
+          
+  appcast:
+    name: Set Appcast
+    needs: build
+    runs-on: macos-latest
+    env:
+      JOB_TYPE: BUILD
+      SIGNATURE_FOR_SIGNING: ${{ secrets.SIGNATURE_FOR_SIGNING }}
+    if: github.event_name == 'release'
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Get Tag
+        run: echo "TAG_VER=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+      
+      - name: Get Information & Sign
+        run: |
+           echo "DATE=$(date -R)" >> $GITHUB_ENV
```

Comment:
Same for DATE and SIGNATURE really. Why cannot it be e.g.

```sh
TAG_VER=${GITHUB_REF/refs\/tags\//}
DATE=$(date -R)
SIGNATURE=$(./bin/sign_update -s ${{ secrets.SIGNATURE_FOR_SIGNING }} build/Release/MaciASL-${TAG_VER}-RELEASE.dmg)
./bin/generate_appcast -o appcast.xml -s ${{ secrets.SIGNATURE_FOR_SIGNING }} --download-url-prefix https://github.com/acidanthera/MaciASL/releases/download/${TAG_VER}/ build/Release
```

or something similar? Also, this should do `|| exit 1` on failure, no?

Reply:
Fixed and added